scripts/us_census/pep/Population_Estimate_by_ASR/county_2000_2010.py

```diff
@@ -0,0 +1,156 @@
+                '6': '2004',
+                '7': '2005',
+                '8': '2006',
+                '9': '2007',
+                '10': '2008',
+                '11': '2009'
+            }
+            df = df.replace({'Year': _dict})
+            df.insert(6, 'geo_ID', 'geoId/', True)
+            df['geo_ID'] = 'geoId/' + (df['STATE'].map(str)).str.zfill(2) + \
+                (df['COUNTY'].map(str)).str.zfill(3)
+            df['AGEGRP'] = df['AGEGRP'].astype(str)
+            # Replacing the numbers with more understandable metadata headings.
+            _dict = {
+                '0': '0Years',
+                '1': '1To4Years',
+                '2': '5To9Years',
+                '3': '10To14Years',
+                '4': '15To19Years',
+                '5': '20To24Years',
+                '6': '25To29Years',
+                '7': '30To34Years',
+                '8': '35To39Years',
+                '9': '40To44Years',
+                '10': '45To49Years',
+                '11': '50To54Years',
+                '12': '55To59Years',
+                '13': '60To64Years',
+                '14': '65To69Years',
+                '15': '70To74Years',
+                '16': '75To79Years',
+                '17': '80To84Years',
+                '18': '85OrMoreYears'
+            }
+            df = df.replace({"AGEGRP": _dict})
+
+            # Drop unwanted columns.
+            df.drop(columns=['SUMLEV', 'STATE', 'COUNTY', 'STNAME', 'CTYNAME'],\
+                inplace=True)
+            df = df.drop(columns=[
+                'TOT_POP', 'NH_MALE', 'NH_FEMALE', 'NHWA_MALE', 'NHWA_FEMALE',
+                'NHBA_MALE', 'NHBA_FEMALE', 'NHIA_MALE', 'NHIA_FEMALE',
+                'NHAA_MALE', 'NHAA_FEMALE', 'NHNA_MALE', 'NHNA_FEMALE',
+                'NHTOM_MALE', 'NHTOM_FEMALE', 'H_MALE', 'H_FEMALE', 'HWA_MALE',
+                'HWA_FEMALE', 'HBA_MALE', 'HBA_FEMALE', 'HIA_MALE',
+                'HIA_FEMALE', 'HAA_MALE', 'HAA_FEMALE', 'HNA_MALE',
+                'HNA_FEMALE', 'HTOM_MALE', 'HTOM_FEMALE'
+            ])
+            df['WhiteAloneAgg'] = df['WA_MALE'].astype(int)+\
+                df['WA_FEMALE'].astype(int)
+            df['BlackOrAfricanAmericanAlone'] = df['BA_MALE'].astype(int)+\
+                df['BA_FEMALE'].astype(int)
+            df['AmericanIndianAndAlaskaNativeAlone'] = df['IA_MALE']\
+                .astype(int)+df['IA_FEMALE'].astype(int)
+            df['AsianAloneAgg'] = df['AA_MALE'].astype(int)+\
+                df['AA_FEMALE'].astype(int)
+            df['NativeHawaiianAndOtherPacificIslanderAloneAgg'] = \
+                df['NA_MALE'].astype(int)+df['NA_FEMALE'].astype(int)
+            df['TwoOrMoreRacesAgg'] = df['TOM_MALE'].astype(int)+\
+                df['TOM_FEMALE'].astype(int)
+            df = df.melt(id_vars=['Year','geo_ID' ,'AGEGRP'], var_name='sv' , \
+                value_name='observation')
+            # Changing Names to be more understandable.
+            _dict = {
```

Comment:
Redefining _dict in the same function. Please either use a different name or in these cases you don't even need to store these in local variables.

Reply:
All the _dict were initially created for better readability. However I have moved them inside the replacement function itself.